packages/atlas/src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -555,6 +563,7 @@ export const CreateEditChannelView: FC<CreateEditChannelViewProps> = ({ newChann
             <ImageCropModal
               imageType="cover"
               onConfirm={handleCoverChange}
+              onDelete={handleDeleteCover}
```

Comment:
Maybe we either:
a) make this prop required
b) hide the `Delete` button if this callback is not defined

This way we avoid situations like this in the future